tests/test_C1_verification.py

```diff
@@ -33,6 +38,45 @@ def test_lookup_file_non_existing_file_raises_error():
         C1.lookup_file(file_path=file_name, name="test")
 
 
+def test_check_if_energy_vector_is_defined_in_DEFAULT_WEIGHTS_ENERGY_CARRIERS_pass():
+    # Function only needs to pass
+    C1.check_if_energy_vector_is_defined_in_DEFAULT_WEIGHTS_ENERGY_CARRIERS(
+        "Electricity", "asset_group", "asset"
+    )
+    assert (
+        1 == 1
```

Comment:
need to add a TODO statement here, or remove the useless test

Reply:
But this checks that the function does not terminate / the test passes - isn't that how you test that?

Reply:
I want to see the assertion debug message if it fails, so I will leave it in.

Reply:
when will `1 == 1` ever fail? I am quite sure `int` don't have sexual reproduction or cellular division